### PR TITLE
fix(dm): shellcheck for modules.d/90dm

### PR DESCRIPTION
**Pull request by haraldh**
This pull request changes...

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

**Review by johannbg — APPROVED** [review]

File: modules.d/90dm/dm-shutdown.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ _do_dm_shutdown() {
 
 if command -v dmsetup > /dev/null \
     && [ "x$(dmsetup status)" != "xNo devices found" ]; then
-    _do_dm_shutdown $1
+    _do_dm_shutdown "${1-$final}"
```
Comment (danimo) [inline modules.d/90dm/dm-shutdown.sh:50]: This changes behavior. Document in commit msg / Split out into separate commit?

**Review by danimo — CHANGES_REQUESTED** [review]

File: modules.d/90dm/dm-shutdown.sh (hunk 1 above)
Comment (haraldh) [reply modules.d/90dm/dm-shutdown.sh:50]: yeah, let me revert this .. it was made under false assumptions

Comment (haraldh) [reply modules.d/90dm/dm-shutdown.sh]: reverted

**Review by johannbg — APPROVED** [review]

**Review by danimo — APPROVED** [review]
